### ISSUE TITLE
Question: Combining Pull Requests

### ISSUE THREAD

**Issue by SteveIreland** [opensourcepos/opensourcepos#2048]
Since there doesn't seem to a desire to merge the  multi-pack branch and I hate for it to just be hanging out there forever to go stale I'd prefer to merge the multi-pack branch into the work I'm doing for India GST and deliver it as a single package so that it will be continuously tested as part of the work I'm doing on India GST.

Thoughts?

**Comment by mckaygerhard** [comment]
where's the source code! i dont see any indian GST work here!

**Comment by WebShells** [comment]
Hello @SteveIreland it's a good feature though am not following up on where tge request is stuck...

**Comment by SteveIreland** [comment]
As I said, It's in development.  It won't be made available until it is ready for public perusal.

**Comment by mckaygerhard** [comment]
ah! ok i understand @SteveIreland , but how other can help if cannot see/understand the code?

by example i made the account basic in my cxc brand that @WebShells can take in basic concept to complete or improve their amount due brand.. and if not! well helps others with not enought programing skills (and was very helpfull for some one)

ther's no sense in testing something that cannot touched!

now off topic, what is the meaning of "perusal" i just curious..

**Comment by SteveIreland** [comment]
@WebShells last I heard was that @daN4cat wanted to hold off on the merge until he had time to test the temporary items fix.  That was awhile ago and it's been sitting out there for about a month without be touched or looked at by anyone, so I'd rather assimilate it into a branch that I'm actively working on.

@mckaygerhard perusal is "the action of reading or examining something" and I'm not asking for any reviews on it yet.  That will hopefully be coming sometime within the next month.

**Comment by mckaygerhard** [comment]
thanks for the ilustration of "perusian", but still not take in care my response: i said "ow other can help if cannot see/understand the code?" i remenber that in 3.2.0 many bugs raises.. (and still are there)

now  have the hands in short due i not have 24/7 network.. there's most the people that can help and make something but dont have 24/7 network connection rather those that have the resourses.. but are so busy--

in any case i'll waith the code sharing, and wheantime i make something about async calls in very basic but at leats will work

**Comment by daN4cat** [comment]
I tested the fixes in master and fixed the item_kits form.
I can merge this later but no time to test because I'm going on holidays. So you need to deal with any follow up issue and the like.

**Comment by daN4cat** [comment]
@SteveIreland I state the obvious to you.

**Comment by jekkos** [comment]
@SteveIreland I'll try to have a look at the branch this weekend

**Comment by SteveIreland** [comment]
I apologize for being clueless on this one.  I ended up in the hospital a couple of weeks ago with a neck fracture. Maybe there is some brain damage?

**Comment by WebShells** [comment]
Thats sad... Lol

**Comment by SteveIreland** [comment]
True ... so true.

**Comment by daN4cat** [comment]
Hope you recover soon, sounds awful.

**Comment by mckaygerhard** [comment]
@SteveIreland i not answered inmediatelly due i not have 24/7 internet connection..  some months ago i fracture my foot was not horrible but was very unconfortable

**Comment by SteveIreland** [comment]
I visited the neurosurgeon a couple of days ago and he said there were 4 or 5 places along my spine that suffered some minor damage but there wasn't anything he could do for me.  I was just ecstatic to git rid of the neck brace.  Essentially I have to stop acting like I'm spider man around boat docks and start acting my age ... but where is the fun in that?  @mckaygerhard I can sympathize,  I also did some damage to one of my fingers which is in a splint ...  it makes typing on the keyboard a real pain.  :-)    But I will be  here for awhile ... I'm determined to finish the Inda GST project!!!
